packages/extension-ui/src/components/AccountNamePasswordCreation.tsx

```diff
@@ -141,6 +143,12 @@ function AccountNamePasswordCreation({
     </CustomFooter>
   );
 
+  const nameRelatedInputs = name ? [name, ...name.split(/\s+/)] : [];
```

Comment:
Would it make sense to split by any non-letter character, instead of just white characters? So that e.g. for sth like `joe_black` it'd recognize 2 words in it.
```suggestion
  const nameRelatedInputs = name ? [name, ...name.split(/[^a-zA-Z]+/)] : [];
```